scripts/bump-version.js

```diff
@@ -1,21 +1,47 @@
 #!/usr/bin/env node
 /* eslint-env node */
 
-const sh = require('shelljs')
-const path = require('path')
-
-sh.set('-e')
+const sh = require("shelljs")
+const path = require("path")
 
+sh.set("-e")
 const lernaConfigPath = path.join(__dirname, '..', 'lerna.json')
 const rootPkgPath = path.join(__dirname, '..', 'package.json')
+const rootPkgLockPath = path.join(__dirname, '..', 'package-lock.json')
 
 const main = () => {
     sh.exec(`lerna version --no-push --no-git-tag-version --yes ${process.argv.slice(2).join(' ')}`)
     sh.exec(`npm install`)
     const lernaConfig = JSON.parse(sh.cat(lernaConfigPath))
     const rootPkg = JSON.parse(sh.cat(rootPkgPath))
+    const rootLockPkg = JSON.parse(sh.cat(rootPkgLockPath))
+
+    // find all monorepo packages, look inside each package json, find peerDependency that is a monorepo package
+    // and update it with a new version
+    sh.exec('npm run --silent lerna -- list --all --json', {silent: true}, function(code, stdout) {
```

Comment:
```suggestion
    sh.exec('lerna list --all --json', {silent: true}, function(code, stdout) {
```
Based on line 13, it looks like you can just invoke lerna directly.
(If that's not true, line 13 should be updated!)

Reply:
You are right, no need to use npm run here. I've removed it. Thanks for catching that